src/ajax/ajax.js

```diff
@@ -43,10 +44,33 @@ define([ "shoestring" ], function(){
 			return;
 		}
 
+		// create parameter string from data object
+		if( settings.data !== null ){
```

Comment:
What happens when `settings.data` is undefined? I noticed both new tests assume data.